content/swift/concepts/classes/classes.md

```diff
@@ -0,0 +1,110 @@
+---
+Title: 'Classes'
+Description: 'Defines characteristics and behaviors of an object.'
```

Comment:
For General entries, I think we can start with the noun:

`Description: 'Classes in Swift defines characteristics and behaviors of an object.'`